src/distributed_ls/Euclid/_hypre_Euclid.h

```diff
@@ -437,24 +439,20 @@ extern HYPRE_Int  ref_counter; /* for internal use only!  Reference counter
  * macros defined in "macros_dh.h"
  */
 extern bool  errFlag_dh;
-extern void  setInfo_dh(const char *msg,const char *function,const char *file, HYPRE_Int line);
-extern void  setError_dh(const char *msg,const char *function,const char *file, HYPRE_Int line);
+extern void  setInfo_dh(const char *msg, const char *function, const char *file, HYPRE_Int line);
+extern void  setError_dh(const char *msg, const char *function, const char *file, HYPRE_Int line);
 extern void  printErrorMsg(FILE *fp);
 
 #ifndef hypre_MPI_MAX_ERROR_STRING
 #define hypre_MPI_MAX_ERROR_STRING 256
 #endif
 
 #define MSG_BUF_SIZE_DH MAX(1024, hypre_MPI_MAX_ERROR_STRING)
-#if defined(HYPRE_USING_RAJA) || defined(HYPRE_USING_KOKKOS) || defined(HYPRE_USING_CUDA)
-static char  msgBuf_dh[MSG_BUF_SIZE_DH];
```

Comment:
I don't recall why I needed to change this for RAJA, KOKKOS or CUDA, so I don't know why we don't need it now. Maybe the C++ junk is gone?

Reply:
I'm not sure either. However, those lines were deleted in `_hypre_Euclid.h` because I reran the `headers` script and those lines were not present in `euclid_common.h`, which is the file that defines `MSG_BUF_SIZE_DH MAX`

Reply:
It was my mistake that I modified `_hypre_Euclid.h` directly without using `headers`.

Reply:
If you ran the regression tests and passed, I guess we may not need this #if, since we now compile Euclid with C compilers instead of nvcc (C++).

Reply:
Great, Ruipeng! I ran machine-tux and it passed. Now, I'm running machine-lassen as well